Example/Podfile

```diff
@@ -5,7 +5,7 @@ platform :ios, '8.0'
 use_frameworks!
 
 target 'GiniVision_Example' do
-    pod 'Gini-iOS-SDK', :path => '../../gini-sdk-ios/'
+    pod 'Gini-iOS-SDK', :git => 'git@github.com:gini/gini-sdk-ios.git', :branch => 'develop'
```

Comment:
Cool, just make sure to add a version, not a branch before merging `multipage`

Reply:
Added an issue in Github so I won't forget about it before release.